frontend/src/metabase/entities/dashboards.js

```diff
@@ -84,13 +84,16 @@ const Dashboards = createEntity({
       Dashboards.actions.update(
         { id },
         { archived },
-        undo(opts, "dashboard", archived ? "archived" : "unarchived"),
+        undo(opts, "dashboard", archived ? "trashed" : "restored"),
```

Comment:
same question as above about localizing

Reply:
resolved using the function i inlined a few comments before this one